mkdocs-base.yml

```
@@ -10,29 +10,34 @@ nav:
     - Context: ./introduction/context.md
     - Acknowledgements: ./introduction/acknowledgements.md
   - Language specification:
+    - About the grammar sections: ./language_specification/about_the_grammar_sections.md
     - Structure:
       - General structure: ./language_specification/structure/general_structure.md
       - File extension: ./language_specification/structure/file_extension.md
       - Case sensitivity: ./language_specification/structure/case_sensitivity.md
-      - Whitespace & Comments: ./language_specification/structure/whitespace_comments.md
-      - Identifiers: ./language_specification/structure/identifiers.md
-      - Reserved keywords: ./language_specification/structure/reserved_keywords.md
+    - Tokens:
+      - Tokens: ./language_specification/tokens/tokens.md
+      - White space: ./language_specification/tokens/white_space.md
```

Whitespace is one word and also the comments are part of this section.
```suggestion
      - Whitespace & Comments: ./language_specification/tokens/whitespace_comments.md
```
The filename will need to be adjusted accordingly.

---

Well, the C++ specification uses `white space`. You can also see it accepted in [Wikipedia](https://en.wikipedia.org/wiki/White_space). And the point of not calling that entry `Whitespace & Comments`, apart from being more concise, is because in the Tokens entry I'm introducing the term as `Blanks, tabs, and comments (collectively, "white space")`.

---

Changed the name to `whitespace`. Whitespaces are not tokens but they have an entry in the Token section, as they are a residue of the lexical analysis (whose task is to convert input text into tokens).

---

Thanks for the explanation!

I do want to have the 'comments' as part of the section title, because readers should be able to find it easily (and we will not have a separate 'Tokens' section with text explaining what we understand 'whitespaces' to be. 

So the section on the whitespace and the comments should be either named "Whitespaces & Comments" or "Whitespaces and Comments"

---

Done!